src/test/resources/graphics/scenery/tests/examples/volumes/VDIRenderer.comp

```diff
@@ -0,0 +1,2183 @@
+#version 450
```

Comment:
This needs to be cleaned up too

Reply:
This is a duplicated file removed in 8188ce95dce32d75e808c688d145dc39c3b1081d